### DISCUSSION TITLE
static var not being reconigized

### DISCUSSION THREAD

**Discussion by NotTheCoderGb** [HaxeFlixel/flixel#2320] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/84812950/121898554-7f4efe00-ccf1-11eb-9596-27dcc3c3cddc.png)

h e l p

**Comment by RichardBray** [comment, marked as answer]
You need to put the static var inside a class. If you want to use a static variable outside of the class you need to get rid of the static keyword. Maybe the inline one as well 🤔 

https://try.haxe.org/#E0156524

**Comment by TheHENOOB** [comment]
All variables should be inside of a class or function
